### ISSUE TITLE
Full test dataset including all 100 subjects?

### ISSUE THREAD

**Issue by kyleolsz** [khan9048/Group_C3I-Team---3D-Synthetic-Data-Projects-group#4]
Hi,

    I checked the link to the provided dataset, which said that it was supposed to include 100 subjects including a mix of male and female subjects in separate zip files. It looks like there are only 2 zip files, both of which appear to contain images for the female subjects. Are the files containing the data for the male subjects available somewhere?
    
    Thanks!

**Comment by khan9048** [comment]
Hi,

The full datasets can be found here https://ieee-dataport.org/documents/c3i-synthetic-face-depth-dataset
________________________________
From: kyleolsz ***@***.***>
Sent: Wednesday, November 9, 2022 5:58 AM
To: khan9048/Facial_depth_estimation ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [khan9048/Facial_depth_estimation] Full test dataset including all 100 subjects? (Issue #4)

EXTERNAL EMAIL: This email originated outside the University of Galway. Do not open attachments or click on links unless you believe the content is safe.
RÍOMHPHOST SEACHTRACH: Níor tháinig an ríomhphost seo ó Ollscoil na Gaillimhe. Ná hoscail ceangaltáin agus ná cliceáil ar naisc mura gcreideann tú go bhfuil an t-ábhar sábháilte.


Hi,

I checked the link to the provided dataset, which said that it was supposed to include 100 subjects including a mix of male and female subjects in separate zip files. It looks like there are only 2 zip files, both of which appear to contain images for the female subjects. Are the files containing the data for the male subjects available somewhere?

Thanks!

**Comment by kyleolsz** [comment]
Yes, that's the site I used for the datasets mentioned on the page. But I only see 2 files: FaceDepthSynth_Female_Part1.zip and FaceDepthSynth_Female_Part2.zip. It looks like they only have female subjects from the ones I checked, and less than 100 subjects in total.

There are about 70 sub-folders with subject IDs, but some of them are empty.

Are these all the subjects that are available? Or are there more dataset files somewhere else that I'm missing?

Thanks again!

**Comment by khan9048** [comment]
Yes those two files are the ones but the IEEE data port have some issues to upload the rest of the data, I will ley you know once complete.
